### PR TITLE
[TOOLS-2447] Block a task with subtasks on a feature

### PR DESCRIPTION
**Pull request by lucasfantacuci**

### CODE REVIEW

File: application/src/test/java/objective/taskboard/it/CardDetailsIT.java [hunk 1]
```diff
@@ -23,6 +23,66 @@ public void whenDescriptionIsChanged_ShouldKeepNewValueAfterReloadPage() {
...
+                .assertIsNotBlocked()
+                .closeDialog();
+
+        mainPage.issue("TASKB-680")
+                .assertHasBlockedIcon(false)
+                .click()
+                .issueDetails()
+                .assertIsNotBlocked();
+    }
+
```
Comment (mauricio-frontend) [inline application/src/test/java/objective/taskboard/it/CardDetailsIT.java]: Make sense a test to guarantee that a subtask is blocked when your parent is blocked.

File: application/src/main/resources/templates/index.html [hunk 2]
```diff
@@ -162,7 +162,8 @@
 
     var TYPE_FIELD = {
         TEXTAREA: "textarea",
-        VERSIONS: "versions"
+        VERSIONS: "versions",
+        BOOLEAN_CHECKBOX: "boolean_checkbox"
```
[the remark this answers is not in the document]
Comment (lucasfantacuci) [reply application/src/main/resources/templates/index.html]: Actually there are two types of check box at the frontend.
A boolean check box: You can specify if a specific value is true or false;
A list check box: you can select some items from a list.

you may ask so, "Why was it just added the boolean checkbox ?" 
YAGNI - You Ain't Gonna Need It.

There is no such case that a LIST_CHECKBOX is needed so it was not added.

So i think the name "BOOLEAN_CHECKBOX" is fair enought.

File: core/src/main/resources/static/elements/taskboard/fields-required-modal.html [hunk 3]
```diff
@@ -194,6 +202,10 @@
                     return type == TYPE_FIELD.VERSIONS;
                 },
 
+                _isBooleanCheckbox: function(type) {
```
[the remark this answers is not in the document]
Comment (lucasfantacuci) [reply core/src/main/resources/static/elements/taskboard/fields-required-modal.html]: The same thing as https://github.com/objective-solutions/taskboard/pull/595#discussion_r283605091

File: application/src/main/resources/templates/index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (mauricio-frontend) [reply application/src/main/resources/templates/index.html]: Ok, in this case is better maintain the name. Tks

**Pull request merged**